cwf/gateway/tools/uesim_cli/main.go

```diff
@@ -232,7 +242,7 @@ func handleTrafficGenCmd(cmd *commands.Command, args []string) int {
 	return 0
 }
 
-func getConfiguredSubscribers() ([]*protos.UEConfig, error) {
+func getConfiguredSubscribers(params ...string) ([]*protos.UEConfig, error) {
```

Comment:
I am having a hard time understanding this function. Maybe it would be good you to separate this into smaller functions to see the different cases. Also you should add some comments for what params are

Reply:
Added comments for clarity.